### PR TITLE
Add a couple of docstrings

### PR DESCRIPTION
**Pull request by timholy**
I find it helpful if the parent module has a brief API summary, so I don't have to visit the webpage or build the docs. That said, perhaps the demo is too much and I should move it elsewhere?

### CODE REVIEW

**Review by ianshmean — COMMENTED** [review]
Absolutely. Thanks!

File: src/VideoIO.jl [hunk 1]
```diff
@@ -159,4 +159,27 @@ function __init__()
...
+
+```julia
+using VideoIO
+r = openvideo(filename)
+img = read(r)
+while !eof(r)
+    read!(r, img)
+end
+```
+"""
```
[the remark this answers is not in the document]
Comment (timholy) [reply src/VideoIO.jl:182]: As an addition or replacement?

[the remark this answers is not in the document]
Comment (ianshmean) [reply src/VideoIO.jl:182]: I think as an addition. Users don't always want the whole video in memory

[the remark this answers is not in the document]
Comment (ianshmean) [reply src/VideoIO.jl:182]: Merged as it didn't need to wait for that change. Thanks!

**Pull request merged**